SDK/src/NDK/Widgets/TextAreaWidget.cpp

```diff
@@ -3,14 +3,15 @@
 // For conditions of distribution and use, see copyright notice in Prerequisites.hpp
 
 #include <NDK/Widgets/TextAreaWidget.hpp>
-#include <Nazara/Core/Unicode.hpp>
 #include <NDK/Components/GraphicsComponent.hpp>
 #include <NDK/Components/NodeComponent.hpp>
+#include <Nazara/Core/Unicode.hpp>
 
 namespace Ndk
 {
 	TextAreaWidget::TextAreaWidget(BaseWidget* parent) :
 	BaseWidget(parent),
+	m_acceptedCharacters([] (char32_t) { return true; }),
```

Comment:
In my opinion, it would be better to use `nullptr` when there's no filter, and to handle it in the check.